src/metabase/automagic_dashboards/populate.clj

```diff
@@ -43,9 +44,22 @@
         :location "/")
       (create-collection! "Automatically Generated Dashboards" "#509EE3" nil nil)))
 
-(def colors
+(defn- get-whitelabeled-colors
```

Comment:
```suggestion
(defn- whitelabeled-colors
```

See https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#pure-function-names-should-be-nouns and https://stuartsierra.com/2016/01/09/how-to-name-clojure-functions

Reply:
This function isn't pure though as it depends on `public-settings/application-colors`. Should I make `public-settings/application-colors` an argument instead what do you think?